ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -1,4 +1,5 @@
 /* Copyright (C) 1999-2013 by Peter Eastman
+   Copyright (C) 2016 by Maksim Khramov
```

Comment:
For clarity, I'd suggest that Copyright notifications for edits should start "Changes Copyright..." as your copyright only applies to the code that you've added or changed. I know it's another nitpick, but better to get things figured out before it gets us in trouble...